### PR TITLE
Comments added to MainActivity.java

### PR DESCRIPTION
**Pull request by shamathmika**
Added comments to MAinActivity.java, changed comments to standards in all files and removed unnecessary import statements.

### CODE REVIEW

**Comment by sandarumk** [comment]
https://github.com/systers/PCSA-Android/wiki/PR-Best-Practices
Read the wiki.
The commit message is wrong. You have not only done changes to MainActivity.

**Comment by shamathmika** [comment]
@sandarumk 
Sorry about that. New here. Will make the changes and send a PR again. Thanks!

**Comment by sandarumk** [comment]
I haven't finished my review yet. If you can wait until it is finished, it will avoid you the trouble of changing the PR once again.

**Comment by shamathmika** [comment]
@sandarumk 
Sure.

**Comment by sandarumk** [comment]
There are certain files in a project like .properties file which shouldn't be committed. You have to remove them from your commit too.

**Comment by sandarumk** [comment]
Please do the changes in the remaining classes only. We can leave the other modules to other new contributors. In that way we can limit to one commit for the comments in remaining classes.

**Comment by sandarumk** [comment]
Please do the changes in the remaining classes only. We can leave the other modules to other new contributors. In that way we can limit to one commit for the comments in remaining classes. Can you make a new issue for removing unused imports. So we can have a separate PR and a commit for that which makes this clean.

**Comment by shamathmika** [comment]
@sandarumk 
Okay. Thank you :) I'll make the changes and send a PR asap.

**Comment by shamathmika** [comment]
@sandarumk 
Is this right?
https://github.com/systers/PCSA-Android/pull/281/commits/5e63937537888de760dc5cbf8cad80bb1c6354ba

**Comment by sandarumk** [comment]
Did you go through all the remaining classes? Only the MainActivity lacks comments?

**Comment by shamathmika** [comment]
@sandarumk 
Yes. All the classes not included in any sub-modules have comments.

**Comment by sandarumk** [comment]
I found that the classes like MainActivityFragment and Constants do not have comments in the standards way. (Those were the only classes I went through after reading your comment.) Please go through the remaining classes again and add comments.

**Comment by shamathmika** [comment]
@sandarumk 
Hey, sorry! They had comments, so I thought it was expected to let them be.
Anyway, made the changes.
Is this it? 
8ad3ac700959afd43f4330d26994c79a8231eb7e

**Comment by sandarumk** [comment]
1. Too much commits
2. Your PR still contains the app.iml file

File: app/src/main/java/com/peacecorps/pcsa/MainActivity.java [hunk 1]
```diff
@@ -46,6 +46,14 @@
 import java.util.LinkedList;
 import java.util.List;
 
+/**
+ * Activity used to load the right placeholder fragment based on availability of instance from
```
Comment (sandarumk) [inline app/src/main/java/com/peacecorps/pcsa/MainActivity.java]: Comment too long.

**Review by sandarumk — COMMENTED** [review]
Too much commits  and till contains the app.iml file

**Comment by sandarumk** [comment]
@shamathmika Any update on this?

**Comment by shamathmika** [comment]
@sandarumk 
Hi! I am extremely sorry. Is it okay if I do this after December 5th?

**Comment by sandarumk** [comment]
Sure.

**Comment by sandarumk** [comment]
@shamathmika any updates?

**Comment by shamathmika** [comment]
@sandarumk hi. I am having too many troubles with committing.

**Comment by sandarumk** [comment]
What are the specific problems you are having?
You can also join systers slack group and ask your questions in #firstaide channel. We would be able to help you there. Just for a quick suggestion, have you tried to have a fresh copy and apply your changes?

**Comment by ashu-dadhich** [comment]
@shamathmika rebasing might correct your merge conflict problems. Perhaps @sandarumk can throw more light on it.

**Comment by codingblazer** [comment]
Hey @top-gun007 , Just rebasing won't resolve merge conflicts.
@shamathmika  Fetch upstream and then rebase, you will find all the Conflicting statements in your Android studio. Decide on what to keep (without erasing anybody else's work) . Then add Main Activity and continue with rebase. This will do it. :-)

**Comment by ashu-dadhich** [comment]
Thats what I meant @codingblazer. Thank you for elaborating it.

**Comment by shamathmika** [comment]
Hey!
Sorry for the late reply, I did the whole procedure again.
Is this good?
4107e534acc6e491a142baf7bafe9a5b9253d71a

**Comment by ashu-dadhich** [comment]
The following points might help:

- reference your issue in the pr so that its easy for the mentors to know regarding what issue the pr was.

- squash your commits if they belong to same issue unless stated otherwise. use (git rebase -i yourbranch~n).

- this one has merge conflicts resolve them. A bit of google search might do it.

-   remove the necessary files as suggested above(app.iml etc.)

**Comment by sandarumk** [comment]
@shamathmika You should go with @top-gun007's suggestions.

**Comment by sandarumk** [comment]
@shamathmika are you still on this? Of you want to start this fresh, you can close this PR and send a new one.

**Comment by ashu-dadhich** [comment]
@sandarumk I think she might have deleted the repo corresponding to this PR thats why it is showing "Unknown repository" at the top. If she is not working on it then I would like to correct it.

**Comment by shamathmika** [comment]
@sandarumk I had sent another PR. I don't think that had any problems. I had even shared it here (4107e53). If that is not right, I think you should let someone else do it since I'm taking way too long figuring it out.

**Comment by sandarumk** [comment]
@shamathmika first of all there is no other PR from you. This is the only PR I found from you in the list. What you have mentioned here is the commit. And yes your mentioned commit has no problems.  This PR contains https://github.com/systers/FirstAide-Android/pull/281/files these changes. 
There is no hurry of you really want to do this. What you can do is fetch and rebase your fork, make a new branch and apply your latest commit by cherrypicking. Then send a PR to systers/develop from that branch. 
You can do a basic google search for fetch, rebase and cherrypicking. This is the easiest solution I can think of. (Some other more easier solutions can exist).

**Comment by sandarumk** [comment]
If you find cherrypicking hard, just copy your comments in your new branch. Do not copy the whole file. It might have updated from your existing copy.

**Comment by sandarumk** [comment]
Closing this currently since no response. If you are still on it, pease send a new PR

**Pull request closed without merging**